modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java

```diff
@@ -240,7 +240,8 @@ else if (name.equals(AccountRoleConstants.ROLE_NAME_SUPPLIER)) {
 					"MANAGE_COMMERCE_ORDER_PAYMENT_TERMS",
 					"MANAGE_COMMERCE_ORDER_PRICES",
 					"MANAGE_COMMERCE_ORDER_RESTRICTED_NOTES",
-					"MANAGE_COMMERCE_ORDER_SHIPPING_OPTIONS"
+					"MANAGE_COMMERCE_ORDER_SHIPPING_OPTIONS",
+					"MANAGE_COMMERCE_ORDERS"
```

Comment:
MANAGE_COMMERCE_ORDERS why is this needed? can't we manage it with the custom logic instead? This is givin him access to manage any order

Reply:
Opted to use localService instead here: https://github.com/liferay-commerce/liferay-portal/pull/3912/commits/77f4ce8b5e1a4b9db76f9477f54b10a8817a5eb5

Reply:
In the shipment view, the permission check is done on the CommerceOrder of the address (This will always be the parent commerecOrder, the original non-split) which Supplier does not have permission to. Opted to use localService instead for all address related fetch calls in shipment-web: https://github.com/liferay-commerce/liferay-portal/pull/3912/commits/3f7a41c86f21e3bf50e28cffb7cf1b08b58a0fd6